gcs-action/action.yml

```diff
@@ -0,0 +1,26 @@
+name: 'GCS Action'
+description: 'Enable easy upload of files and directories to GCS'
+author: 'GoogleCloudPlatform'
+inputs:
+  service_account_key:
+    description: 'GCP service account key'
+    required: true
+  folder-to-upload:
+    description: 'Folder to be uploaded'
+    required: false
+  file-to-upload:
+    description: 'File to be uploaded'
+    required: false
+  bucket-name:
+    description: 'GCS Bucket Name'
+    required: true
+  object-prefix:
```

Comment:
What is this used for?

Reply:
This was intended to be the prefix; now it has been pulled in the single input of form destination: my-bucket/prefix